src/Helper/MyParcelCollection.php

```diff
@@ -511,6 +511,81 @@ public function clearConsignmentsCollection()
         $this->items = [];
     }
 
+    /**
+     * @param int $id
+     * @param string $apiKey
+     *
+     * @return \MyParcelNL\Sdk\src\Helper\MyParcelCollection
+     */
+    public static function find(int $id, string $apiKey): MyParcelCollection
+    {
+        $collection = self::findMany([$id], $apiKey);
+
+        return $collection;
+    }
+
+    /**
+     * @param array $consignmentId
+     * @param string $apiKey
```

Comment:
Misschien even in phpstorm instellen dat $... netjes onder elkaar staan. Zie ook andere plekken waar je typehint (php 7) hebt geprogrammeerd.

Reply:
Done